src/scripts/ecr-login.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+REGION=$(eval echo "${PARAM_REGION}")
```

Comment:
Is an `eval` here needed? It looks like this might be attempting parameter expansion. Example: https://github.com/CircleCI-Public/github-cli-orb/blob/081b9997904ebe3758104139065945c22cb53b16/src/scripts/configure.sh#L3

Applies to the lines below